README.rst

```diff
@@ -34,6 +34,14 @@ Finally, the platform can be run as usual:
 
     tutor local quickstart
 
+Configuration
+-------------
+
+- ``ENABLE_CODEJAIL_REST_SERVICE`` (default: ``True``)
+- ``CODE_JAIL_REST_SERVICE_HOST``: (default: ``http://{{ CODEJAIL_HOST }}:8550``)
+- ``CODE_JAIL_REST_SERVICE_CONNECT_TIMEOUT``: (default: ``0.5``)
+- ``CODE_JAIL_REST_SERVICE_READ_TIMEOUT``: (default: ``3.5``)
```

Comment:
These variables are the ones that end up in the LMS/CMS, but aren't really the configuration variables of the plugin that you put in your `config.yaml`. Those variables are defined [here](https://github.com/eduNEXT/tutor-contrib-codejail/blob/main/tutorcodejail/plugin.py#L13-L20). Remember to prepend them with the plugin name in uppercase (e.g. `CODEJAIL_HOST`)

Reply:
Oh good! Didn't know that, thank you so much. Then it's just CODEJAIL_HOST and CODEJAIL_SECRET_KEY?

Reply:
It should be all of them. For example `DOCKER_IMAGE` is useful for people that push to a different registry.

But it seems some of them are not really being used? Like `SECRET_KEY` (I think this is a remanent from where it was a Django project instead of flask).